jenkins/ci.suse.de/templates/cloud-socok8s-ansible-lint.yaml

```diff
@@ -0,0 +1,20 @@
+- job-template:
+    name: '{name}-ansible-lint'
+    project-type: multibranch
+    periodic-folder-trigger: 5m
+    number-to-keep: 30
+    days-to-keep: 30
+    script-path: Jenkinsfile.lint
+    scm:
+      - github:
+          repo: '{repo-name}'
+          repo-owner: '{repo-owner}'
+          credentials-id: '{repo-credentials}'
+          branch-discovery: no-pr
+          discover-pr-forks-strategy: merge-current
```

Comment:
I think `current` is enough .AFAIU, `merge-current` results in rechecks whenever the target branch (in our case master) changes. But for linting, that is not needed imo.

Reply:
done